### ISSUE TITLE
About lane classification

### ISSUE THREAD

**Issue by wsliu1999** [IrohXu/lanenet-lane-detection-pytorch#1]
Dear author, have you try about lane classification using LaneNet? We had a new project about lane detection and classification, so have you process some multi-class lane before? If so, could you share some experience? Thanks a lot for your code! Your work is awesome!

**Comment by IrohXu** [comment]
It depends on the dataset provider. Since tusimple dataset do not label multi-class lane such as solid lane, broken lane, I only use a binary output in this model. One trick to detect multi-class lane is to change the output of the binary branch to a one-hot format. For example: change 1x224x224 output to 5x224x224 output and use softmax. And also, you need have a multi-class labeled dataset.

Iroh
